packages/cardpay-reward-programs/cardpay_reward_programs/payment_tree.py

```diff
@@ -145,7 +145,7 @@ def as_arrow(self):
                 columns["proof"].append(self.get_hex_proof(leaf))
                 columns["explanationId"].append(explanation_id)
                 columns["explanationData"].append(
-                    [(k, str(v)) for k, v in payment["explanationData"].items()]
+                    json.dumps(payment["explanationData"], separators=(",", ":"))
```

Comment:
I think the cutom separators is just removing the spaces - if this is required for s3select can you add a comment and a test? That's the kind of subtle regression we could see at some point. If it works anyway it'd be good to use just json.dumps with its defaults.